prov/util/src/util_attr.c

```diff
@@ -444,6 +444,7 @@ int ofi_check_mr_mode(uint32_t api_version, uint32_t prov_mode,
 			return -FI_ENODATA;
 		}
 	} else {
+	    return 0;
```

Comment:
Sorry, it is from copy-paste from the draft version of the development :) it must be removed